tls/s2n_psk.c

```diff
@@ -135,9 +133,9 @@ S2N_RESULT s2n_psk_parameters_init(struct s2n_psk_parameters *params)
 
 static S2N_RESULT s2n_psk_offered_psk_size(struct s2n_psk *psk, uint32_t *size)
 {
-    *size = sizeof(uint16_t)    /* identity size */
-          + sizeof(uint32_t)    /* obfuscated ticket age */
-          + sizeof(uint8_t)     /* binder size */;
+    *size = sizeof(uint16_t)   /* identity size */
+            + sizeof(uint32_t) /* obfuscated ticket age */
+            + sizeof(uint8_t) /* binder size */;
```

Comment:
I thought it was supposed to line up comments like these? Did we turn that off?

Reply:
No. These comments aren't part of an expression. Evidently it doesn't handle this, rather odd case. On the bright side a manual fix works no problem.